test/blackbox-tests/test-cases/cross-compilation-install.t/dune-project

```diff
@@ -0,0 +1,15 @@
+(lang dune 3.2)
+
+(using dune_site 0.1)
+
+(generate_opam_files true)
```

Comment:
Why do we need to generate opam files or the dune_site feature to reproduce this issue?

Reply:
we don't. this was from an earlier iteration. I'll improve.